network-api/networkapi/templates/fragments/footnotes.html

```diff
@@ -12,9 +12,9 @@ <h3 class="footnotes-title">{% trans "Footnotes" %}</h3>
           <a href="#footnote-source-{{ forloop.counter }}" class="footnote-number mr-3" aria-label="{% trans 'Back to content' %}">
             [{{ forloop.counter }}]
           </a>
-          <a href="#footnote-source-{{ forloop.counter }}" class="footnote-text" aria-label="{% trans 'Back to content' %}">
+          <div href="#footnote-source-{{ forloop.counter }}" class="footnote-text" aria-label="{% trans 'Back to content' %}">
```

Comment:
```suggestion
          <div class="footnote-text">
```

Reply:
Haha you mean <div>'s dont have hrefs? 👀 Thanks for catching this!